pinot-segment-local/src/main/java/org/apache/pinot/segment/local/realtime/converter/ColumnDescriptionsContainer.java

```diff
@@ -0,0 +1,48 @@
+package org.apache.pinot.segment.local.realtime.converter;
+
+import java.util.List;
+
+
+public class ColumnDescriptionsContainer {
+  private final String _sortedColumn;
+  private final List<String> _invertedIndexColumns;
+  private final List<String> _textIndexColumns;
+  private final List<String> _fstIndexColumns;
+  private final List<String> _noDictionaryColumns;
+  private final List<String> _varLengthDictionaryColumns;
+
+  public ColumnDescriptionsContainer(String sortedColumn, List<String> invertedIndexColumns,
+      List<String> textIndexColumns, List<String> fstIndexColumns, List<String> noDictionaryColumns,
+      List<String> varLengthDictionaryColumns) {
+    _sortedColumn = sortedColumn;
+    _invertedIndexColumns = invertedIndexColumns;
+    _textIndexColumns = textIndexColumns;
+    _fstIndexColumns = fstIndexColumns;
+    _noDictionaryColumns = noDictionaryColumns;
+    _varLengthDictionaryColumns = varLengthDictionaryColumns;
+  }
+
+  public String getSortedColumn() {
+    return _sortedColumn;
+  }
+
+  public List<String> getInvertedIndexColumns() {
+    return _invertedIndexColumns;
+  }
+
+  public List<String> getTextIndexColumns() {
+    return _textIndexColumns;
+  }
+
+  public List<String> getFstIndexColumns() {
+    return _fstIndexColumns;
+  }
+
+  public List<String> getNoDictionaryColumns() {
```

Comment:
Seems like this method is not being used anywhere but it should ideally be used.

Reply:
Yeah. I wasn't sure why that is the case. It is being passed into `RealtimeSegmentConverter`. But even there, the member variable `_noDictionaryColumns` was unused. So, not sure why this exists. I didn't want to change any functionality in a refactoring PR. So, left it as is.